src/include/session.h

```diff
@@ -59,6 +59,16 @@ struct __wt_prefetch {
     uint64_t prefetch_skipped_with_parent;
 };
 
+/*
+ * WT_HELPER_ERROR --
```

Comment:
I suggest we update these names: I've proposed `__wt_error_info`/`WT_ERROR_INFO` in the project slack channel, and had no objections so far, so I think we should go with those names for now at least.

Reply:
[Renamed WT_HELPER_ERROR to WT_ERROR_INFO](https://github.com/wiredtiger/wiredtiger/pull/11327/commits/0531dd63d99a6a37bad4085d1d5fd03fb9402987)
Done in this commit